dm/dm/config/task.go

```diff
@@ -324,19 +324,19 @@ type SyncerConfig struct {
 	AutoFixGTID bool `yaml:"auto-fix-gtid" toml:"auto-fix-gtid" json:"auto-fix-gtid"`
 	EnableGTID  bool `yaml:"enable-gtid" toml:"enable-gtid" json:"enable-gtid"`
 	// deprecated
-	DisableCausality bool `yaml:"disable-detect" toml:"disable-detect" json:"disable-detect"`
-	SafeMode         bool `yaml:"safe-mode" toml:"safe-mode" json:"safe-mode"`
+	DisableCausality       bool   `yaml:"disable-detect" toml:"disable-detect" json:"disable-detect"`
+	SafeMode               bool   `yaml:"safe-mode" toml:"safe-mode" json:"safe-mode"`
+	SafeModeResumeDuration string `yaml:"safe-mode-resume-duration" toml:"safe-mode-resume-duration" json:"safe-mode-resume-duration"`
```

Comment:
Can it be named `safe-mode-duration`? Or the `resume` means it only takes effect when resuming not starting?

Reply:
It seems `safe-mode-duration` is better to meet demand. LGTM.

Confirm later.